frontend/src/metabase/admin/settings/auth/components/AuthCard/AuthCard.unit.spec.tsx

```diff
@@ -79,11 +79,10 @@ describe("AuthCard", () => {
   });
 });
 
-const getSetting = (opts?: Partial<AuthSetting>): AuthSetting =>
-  createMockSettingDefinition({
-    value: false,
-    ...opts,
-  }) as AuthSetting;
+const getSetting = (opts?: Partial<AuthSetting>): AuthSetting => ({
```

Comment:
there's no point in having a mock here, it's a different type and this is the only place we need to mock it